Dockerfile

```diff
@@ -1,6 +1,6 @@
 ######################################################################
 # Establish a common builder image for all golang-based images
-FROM docker.io/library/golang:1.19 AS golang-builder
+FROM golang:1.19 AS golang-builder
```

Comment:
podman gets upset if you don't specify the actual registry, docker assumes it is docker.io.